test/e2e-v2/cases/mysql/prometheus-mysql-exporter/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    extends:
+      file: ../../../script/docker-compose/base-compose.yml
+      service: oap
+    environment:
+      SW_OTEL_RECEIVER: default
+      SW_OTEL_RECEIVER_ENABLED_OC_RULES: mysql
+    ports:
+      - 12800
+  mysql:
+    image: mysql:5.7
+    networks:
+      - e2e
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+  mysql-service:
+    image: prom/mysqld-exporter
```

Comment:
suggest giving a specific exporter version, like `image: prom/mysqld-exporter:xxx.xx`

Reply:
Which is the version of the showcase using? We need to consider keeping them consistent, at least compatible.

Reply:
It's a new one, we don't have `mysqld-exporter` before

Reply:
I was trying to mean the OTEL, not this. 
And yes, we should keep specific version here.

Reply:
oh, that is `otel/opentelemetry-collector:0.50.0`